### PR TITLE
tgui human inventory

### PR DESCRIPTION
**Pull request by Luxizzle**
## About the PR 

See #6678 & #6144

I was big idiot and removed repo

## Why's this needed? 

Begone jank

image

## Changelog

```changelog
(u)Luxizzle
(*)TGUI Human Inventory
```

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
I wonder how cursed/performant it would be to load icons for all of these...

**Comment by Luxizzle** [comment]
> I wonder how cursed/performant it would be to load icons for all of these...

I've thought about it, but I'd like feature parity first before adding anything new

**Comment by flrsh** [comment]
This looks very nice, thank you for all your TGUI stuff!

**Comment by Luxizzle** [comment]
!merge_upstream

**Comment by Luxizzle** [comment]
!merge_upstream

File: code/datums/inventory.dm [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		MAKE_SLOT("slot_r_store", src.human.r_store),
+	)
+
+	. = list(
+		"name" = src.human.name,
+		"slots" = slots,
+		"handcuffed" = src.human.hasStatus("handcuffed"),
+		"internal" = src.human.internal != null,
+		"canSetInternal" = istype(src.human.wear_mask, /obj/item/clothing/mask) && istype(src.human.back, /obj/item/tank) && !src.human.internal
+	)
```
Comment (ZeWaka) [inline code/datums/inventory.dm:51]: if `update_inv()` exists why can't this all be static and it be refreshed by that

Comment (Luxizzle) [reply code/datums/inventory.dm:51]: I dont think im going to be able to find all the possible places where they might get modified. I can do a pass over it later, but I'd rather keep this to interface changes for now.

I also don't want accidental desyncs between what's in their inventory and what you can see in the interface.

Comment (ZeWaka) [reply code/datums/inventory.dm:51]: fair

**Comment by Luxizzle** [comment]
!merge_upstream

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**